doc/release-notes-1.9.rst

```diff
@@ -74,7 +74,9 @@ Networking
 Bluetooth
 *********
 
-* change description
+* Bluetooth Mesh support (all mandatory features and most optional ones)
+* GATT Service Changed Characteristic support
+* IPSP net-app support
```

Comment:
Might expand on this more, something like:

     * IPSP net-app support: a simplified networking API reducing duplication of common
       tasks an application writer has to go through to connect to the network